docs/reference/storage.md

```diff
@@ -0,0 +1,127 @@
+# Storage
+
+uv persists data in several locations on your system.
+
+## Directory Strategies
+
+uv follows platform conventions for determining where to store different types of data.
+
+Generally, it's best to configure these rather than each uv-specific storage location.
```

Comment:
We should mention that we XDG on Unix specifically here, otherwise it's unclear what platform conventions we mean.